tests/resources/Group23-VIC-Machine-Service-Util.robot

```diff
@@ -150,3 +150,11 @@ Verify VCH List Empty
     ${vchs}=  Run  echo '${OUTPUT}' | jq -r '.vchs[]'
     Log  ${vchs}
     Length Should Be  ${vchs}  0
+
+Get Docker Params API
+    [Arguments]    ${vch}    ${vch_name}
```

Comment:
I think `vch` would be the only valid value for `${vch}` (since that's a hard-coded portion of the API), so I think we can just remove this first argument.